src/screens/DiscoveryScreenV2/elements/Search.js

```diff
@@ -282,7 +283,8 @@ const styles = StyleSheet.create({
     alignSelf: 'center',
     height: dimen.normalizeDimen(33),
     paddingTop: 0,
-    paddingBottom: 0
+    paddingBottom: 0,
+    color: COLORS.white2
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the `input` style has been changed from default to white2. Ensure that this change doesn't affect the visibility of the text, especially if the background is also a light color.
```diff
- color: COLORS.default
+ color: COLORS.white2
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->